lblod/ember-rdfa-editor-lblod-plugins
# Ensure embroider-optimized compatibility

### Overview
This PR includes some changes to ensure `embroider-optimized` compatibility:
- Some changes to the webpack configuration and exporting a webpack config object for host apps in `webpack-config.js`'
- Update `ember-resolver` to 11.0.1
- Addition of `@ember/string` as a dev- and peer-dependency. (see https://deprecations.emberjs.com/v4.x/#toc_ember-string-add-package)
- Include explicit `.js` extension for `process/brower.js` webpack plugin to ensure esmodule compatibility
- Explicitly import ember-velcro component to workaround embroider issue. (see https://github.com/lblod/ember-rdfa-editor/commit/c8e8de883b30b687cccfef8d1bea6d67c1f454e3)

Additionally this PR includes the following changes:
- Addition of `ember-power-select` as a peerdependency. It specifies the `6.x` and `7.x` ranges.

### How to test/reproduce
- Run the `embroider-optimized` scenario (`ember try:one embroider-optimized`) and ensure no errors occur
- Ensure that the dummy application still works as expected.



### Checks PR readiness
